file-formats/uiad/examples/z_aff_example_uiad.uiad.json

```diff
@@ -26,8 +26,7 @@
         "name": "tmParamName",
         "defaultValue": "Hello World!"
       }
-    ],
-    "additionalParameterHandling": "ignored"
```

Comment:
was this supposed to be deleted?

Reply:
For example it better to have this parameter and it is added again. Thanks